_locales/de/messages.json

```diff
@@ -27,6 +27,14 @@
     "message": "Fügt einen Button hinzu, der dich bequem den Kompakt-Modus aktivieren/deaktivieren lässt. Wenn aktiv, werden Tags, zugewiesene Menschen, Anhänge und mehr ausgeblendet. so kannst Du dich auf das konzentrieren, was wirklich wichtig ist.",
     "description": "Description of the Compact Mode feature."
   },
+  "settingsHideAddNewListTitle": {
+    "message": "Verstecke \"Eine weitere Liste hinzufügen\"",
+    "description": "Name of the 'Hide Add New List' feature."
+  },
+  "settingsHideAddNewListDescription": {
+    "message": "Versteckt den \"Eine weitere Liste hinzufügen\"-Button",
```

Comment:
I think something more descriptive would be appropriate. Maybe something along these lines: 

**en**: "This hides the additional column with the "Add new list" button and saves horizontal space. You can still create new lists either by double clicking on an empty area on the board or by using the keyboard shortcut _XY_"

What I want to achieve by this message is 1) that people understand what column is being meant (the image will help with that) and what is the advantage to disabling the button (less horizontal scrolling), and 2) that people can learn how to create a new list even without the button (double click & shortcut).

(I don't know the shortcut right now, but I am pretty sure there is one. The correct shortcut should make it into the message above)